analyzer/codechecker_analyzer/analyzer.py

```diff
@@ -254,12 +256,15 @@ def perform_analysis(args, skip_handler, context, actions, metadata_tool,
             state, _ = data
             metadata_info['checkers'].update({
                 check: state == CheckerState.enabled})
+            enabled_checkers.append(check)
 
         version = config_map[analyzer].get_version(check_env)
         metadata_info['analyzer_statistics']['version'] = version
 
         metadata_tool['analyzers'][analyzer] = metadata_info
 
+    LOG.info("Enabled checkers:\n%s", '\n'.join(enabled_checkers))
```

Comment:
wouldn't a debug level log be enough here? I'm just thinking this list can get really long quickly (just by enabling a few groups), or that is the plan to make it visible?